dts/arm/nordic/nrf51822.dtsi

```diff
@@ -76,7 +76,7 @@
 			status = "okay";
 
 			bt_hci_controller: bt_hci_controller {
-				compatible = "zephyr,bt-hci-ll-sw-split";
+				compatible = "nordic,bt-hci-virtual";
```

Comment:
The binding still uses "zephyr,..." so I can't really see how this will work. Furthermore, since the binding is essentially for `subsys/bluetooth/controller/hci/hci_driver.c` that's (AFAIK) not Nordic specific, rather it can be used with any Lower Link Layer.

Reply:
yes I agree: https://github.com/zephyrproject-rtos/zephyr/pull/74168#discussion_r1637861621

Reply:
Yes. This was a temporary push. Discussing internally how to resolve this in a nice way. It would be nice if could avoid having two instances here